File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        int[] no_of_1 = new int[n+1];
        for (int i = 0;i<n+1;i++){
            no_of_1[i] = no(i);
        }
    return no_of_1;
    }
    private int no(int n){
        int count = 0;
        while (n!=0){
            if((n&1) == 1){
                count += 1;
                n = n >> 1;
            }else{
                n = n >> 1;
            }
            
        }
        return count;
    }
}

File: Data Structures & Algorithms/meeting-schedule-ii/submission-0.py
"""
Definition of Interval:
class Interval(object):
    def __init__(self, start, end):
        self.start = start
        self.end = end
"""

class Solution:
    def minMeetingRooms(self, intervals: List[Interval]) -> int:
        start = []
        end = []
        rooms, count, s, e = 0,0,0,0
     
        for x in intervals:
            start.append(x.start)
            end.append(x.end)

        start.sort()
        end.sort()

        while s < len(start):
            if start[s] < end[e]:
                count += 1
                rooms = max(rooms, count)
                s +=1
            
            else:
                count -= 1
                rooms = max(rooms, count)
                e += 1

        
        return rooms

            
               
        

    